pkg/bundle/create/create.go

```diff
@@ -267,7 +271,15 @@ func CreateDiff(configPath, rootDir, outputDir string, dryRun, insecure bool) er
 	return nil
 }
 
-func prepareArchive(rootDir, outputDir string, segSize int64, files []string) error {
+func prepareArchive(cfg v1alpha1.ImageSetConfiguration, rootDir, outputDir string, files []string) error {
+
+	var segSize int64
+
+	if cfg.ImageSetConfigurationSpec.ArchiveSize != 0 {
```

Comment:
The two suggestions I think we not correct on the diff function which is why they made it in there in the first place.